src/openforms/js/components/admin/form_design/variables/ServiceFetchConfigurationPicker.stories.js

```diff
@@ -0,0 +1,73 @@
+import {FormLogicDecorator} from '../story-decorators';
+import ServiceFetchConfigurationPicker from './ServiceFetchConfigurationPicker';
+
+export default {
+  title: 'Form design/Service Fetch/ServiceFetchConfigurationPicker',
+  decorators: [FormLogicDecorator],
+  component: ServiceFetchConfigurationPicker,
+};
+
+export const Blank = {
+  name: 'Blank',
+
+  // No name supplied, should fall back to "method path (service)"
```

Comment:
Same data, same remark.

Reply:
I moved the comment back to the relevant line of code